SingularityUI/app/actions/api/users.es6

```diff
@@ -0,0 +1,36 @@
+import { buildApiAction, buildJsonApiAction } from './base';
+
+export const FetchUserSettings = buildApiAction(
+  'FETCH_USER_SETTINGS',
+  userId => ({
+    url: `/users/settings?userId=${userId}`,
+    catchStatusCodes: [404] // Remove this once the API doesn't 404 when user has nothing set
+  })
+);
+
+export const UpdateUserSettings = buildJsonApiAction(
+  'UPDATE_USER_SETTINGS',
+  'POST',
+  (userId, settings) => ({
+    url: `/users/settings?userId=${userId}`,
```

Comment:
this might be more of a question for @MattCCS, but why do these endpoints require the userId, but the other two dont? We should standardize on not requiring an explicit userId query param anywhere and let Singularity's authentication mechanism figure out who you are